### DISCUSSION TITLE
How to managing a custom domain in github pages?

### DISCUSSION THREAD

**Discussion by viva-lab** [community/community#63647] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I've added a custom domain earlier in my GitHub page now my domain expired so I no longer able to use that domain, 
problem is whenever I publish a page using github page it takes my old domain as a root directory how can I change this behavior?

**Comment by joy-ahmed** [comment]
Remove Custom Domain from Repository Settings:

Go to your GitHub repository.
Click on the "Settings" tab at the top of the repository.
Scroll down to the "GitHub Pages" section.
You will see the custom domain listed under "Custom domain." Clear the custom domain field.

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
